Lib/defcon/objects/layer.py

```diff
@@ -565,6 +565,9 @@ def save(self, glyphSet, saveAs=False, progressBar=None):
                 if glyphName in glyphSet:
                     glyphSet.deleteGlyph(glyphName)
         glyphSet.writeContents()
+        # XXX The parent FS needs is going to be closed inside Font.save, so
+        # any operations on the glyphSet's SubFS will fail after that.
+        # All the glyph data has been loaded by now; set _glyphSet to None?
         self._glyphSet = glyphSet
```

Comment:
in fact, if I set `self._glyphSet = None` after finishing to save the layer to disk, then the tests for `testForExternalChanges` fail, as these expect the layer object to keep a handle on the glifLib.GlyphSet object.
Maybe we should just accept that when the `Font.ufoFileStructure` is zip, it isn't possible to meaningfully check for external changes.